settings.gradle.kts

```diff
@@ -103,7 +103,7 @@ gradleEnterprise {
 // The HAPI API version to use for Protobuf sources. This can be a tag or branch
 // name from the hedera-protobufs GIT repo.
 val hapiProtoVersion = "0.40.0-blocks-state-SNAPSHOT"
-val hapiProtoBranchOrTag = "change-topic-number-to-topic-id" // hapiProtoVersion
+val hapiProtoBranchOrTag = "07098-add-FileID" // hapiProtoVersion
```

Comment:
Make sure to coordinate with neeha and make sure the branch name you commit this PR with is correct after merging your changes in on the hedera-protobufs side

Reply:
This PR will merge with this branch to avoid the timing issue.
Then a subsequent PR will switch to the merged branch after the protobuf PR is merged.

Reply:
If we are not merging `protobufs` PR first, then @kimbor  in order to avoid breaking develop, 
1. Merge this PR
2. Merge the protobufs PR, make sure this branch `07098-add-FileID` is not deleted. Since develop has this branch.
3. Then open PR to develop changing the branch name on develop to `add-pbj-types-to-state`

or as Jasper suggested in one call we keep the duplicate `fileNum` field in protobufs and remove it later.